addons/purchase/static/src/scss/purchase.scss

```diff
@@ -0,0 +1,48 @@
+.o_purchase_dashboard {
+    flex: 1 0 100%;
+    padding: 10px 10px 18px 10px;
+
+    background-color: $o-view-background-color;
+    position: relative;
+    max-width:100%;
+
+    .o_dashboard_action {
+        cursor: pointer;
+    }
+    
+    .table {
+        margin-bottom: 0;
+        table-layout: fixed;
+        border-spacing: 10px 0px;
+        border-collapse: separate;
+        font-size: 14px;
+
+        td {
```

Comment:
don't use a tag name as selector, the browser has to check all parents element of each td until the top of the dom to see if this style should apply or not

you could tie-break this check early by using a selectore more precise, such as > td (direct child + add a > tr selector too)